plugins/filter/lib/voicecall-filter-plugin.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=/usr
+exec_prefix=${prefix}
+libdir=${prefix}/$$PKGCONFIG_LIB
+includedir=${prefix}/include/voicecall/plugins
+
+Name: voicecall-filter-plugin
+Description: Voice call filtering library
+Version: $$PROJECT_VERSION
```

Comment:
I might just hard-code this to to something to avoid needing to pass the version externally. These can bite back if people start using the project with separate packaging.

Reply:
Hardcode it requires to remember to update it. Which I often forgot. But I went for the create_pc route. It works fine. Thanks for the tips.